sdk/client/package.json

```diff
@@ -10,7 +10,7 @@
 	"scripts": {
 		"build": "yarn types:check && yarn typegen && vite build",
 		"codegen": "graphql-codegen --config codegen.yml",
-		"dev": "vite dev",
+		"dev": "vite dev & vite build --watch",
```

Comment:
could we `run-p` the two commands to avoid a potential orphaned process?

Reply:
Done!